### PR TITLE
fix(tile): avoid `display: none` for hiding selectable tile input field

### LINKED ISSUE #1669: Avoid `display: none` on selectable tile input field
Related: https://github.com/IBM/carbon-components-react/issues/1114, reported by @asthajain1 and @zachstowellibm

## Detailed description

In the Selectable Tile component, the radio button `input` has the CSS rule `display: none` causing an a11y failure. We should explore a different strategy to visually hide the input field

### PR DESCRIPTION
**Pull request by emyarod**
Closes #1669

This PR avoids usage of `display: none` on the selectable tile input field so that the element remains in the DOM and does not cause errors for screen readers

### CODE REVIEW

File: src/components/tile/_tile.scss [hunk 1]
```diff
@@ -312,7 +312,9 @@
   }
 
   .#{$prefix}--tile-input {
-    display: none;
+    float: left;
```
Comment (asudoh) [inline src/components/tile/_tile.scss]: Can we use `@include hidden` here...?

**Review by asudoh — APPROVED** [review]
LGTM, just one comment - Thanks @emyarod!

**Review by joshblack — APPROVED** [review]
Looks great after @asudoh's comment 👍

File: src/components/tile/_tile.scss [hunk 2]
```diff
@@ -312,7 +312,9 @@
   }
 
   .#{$prefix}--tile-input {
-    display: none;
+    @include hidden;
+    opacity: 0;
```
Comment (asudoh) [inline src/components/tile/_tile.scss]: Just 🤔, do we still think we need additional CSS props...?

Comment (emyarod) [reply src/components/tile/_tile.scss]: I think the input field would still be visible if the positioning is off, which is why I left `opacity: 0` in

Comment (emyarod) [reply src/components/tile/_tile.scss]: after I added `@include hidden` and adjusted the positioning, I could still view the radio button behind the tile div but now I think the styles just did not properly load on my local environment when I checked. seems to be fine now

**Pull request merged**